CONTRIBUTING.md

```diff
@@ -4,6 +4,175 @@ First of all, **thank you** for helping with Exercism.io!
 
 We are working to improve this document, and if you find any part of it confusing, or if you can't figure out how to get started with something, then rest assured it's not you, it's us! Please open up a new issue to describe what you were hoping to contribute with, and what you're wondering about, and we'll figure out together how to improve the documentation.
 
+----
+
+# Finding Your Way
+
+## I...
+
+| ... see a problem | ... have an idea | ... would like to help |
+|:---:|:---:|:---:|
+|[![I see a problem](docs/img/a-bug.png)](#i-see-a-problem) | [![I have an idea](docs/img/an-idea.png)](#i-have-an-idea-) | [![I would like to help](docs/img/volunteering-hand.png)](#i-would-like-to-help) |
+
+--
+
+## I see a problem...
+
+| ... on the website | ... with the command-line client (CLI) | ... getting started with a language | ... with a specific exercise |
+|:---:|:---:|:---:|:---:|
+|[![I see a problem on the website](docs/img/globe.png)](#-on-the-website) | [![I see a problem with the CLI](docs/img/prompt.png)](#-with-the-command-line-client-cli) | [![I see a problem getting started with a language](docs/img/target-a-track.png)](#-getting-started-with-a-language) | [![I see a problem with a specific exercise](docs/img/computer-code.png)](#-with-a-specific-exercise) |
+
+
+### ... on the website 
+
+* If you're having trouble figuring out how to do something on the website, jump in the [online support chat](https://gitter.im/exercism/support) and ask there.
+- If the website has a bug, you can help by [filing a bug report](docs/filing-a-bug-report.md) in the [`exercism/exercism.io` repository](https://github.com/exercism/exercism.io/issues/new).
+  - After reporting the problem, if you have the time and the desire, you can help *even more* by [fixing it](#-web-programming).
+  
+-
+
+### ... with the command-line client (CLI)
+
+* If you're having trouble installing or working with the CLI, ask for help in our [online support chat](https://gitter.im/exercism/support).
+- If the CLI has a bug, you can help by [filing a bug report](docs/filing-a-bug-report.md) in the [`exercism/cli` repository](https://github.com/exercism/cli/issues/new).
+  - After reporting the problem, if you have the time and the desire, you can help *even more* by [fixing it](#-go).
+
+-
+
+### ... getting started with a language 
```

Comment:
Ditto.  Thanks for the feedback!!  If you see think of anything else, big or small, please share!